pkg/kubecost/allocationfilter.go

```diff
@@ -0,0 +1,210 @@
+package kubecost
+
+import "github.com/kubecost/cost-model/pkg/log"
+
+// FilterField is an enum that represents Allocation-specific fields that can be
+// filtered on (namespace, label, etc.)
+type FilterField int
```

Comment:
We could make this `type FilterField string` instead, which would give us a much better serialization/deserialization story for pretty printing etc. Thoughts?

The same goes for `FilterOp`.

Reply:
At the cost of slight overhead in terms of string allocations, but I don't think we're going to have a meaningful number of filters allocated at any given time.

Reply:
Updated Field and Op to be strings.